arroyo/backends/kafka/configuration.py

```diff
@@ -49,6 +49,52 @@ def stats_callback(stats_json: str) -> None:
     )
 
 
+def producer_stats_callback(stats_json: str) -> None:
+    stats = json.loads(stats_json)
+    metrics = get_metrics()
+
+    # Extract broker-level int_latency metrics
+    brokers = stats.get("brokers", {})
+    for broker_id, broker_stats in brokers.items():
+        int_latency = broker_stats.get("int_latency", {})
+        if int_latency:
```

Comment:
In what case would this be `None`?

Reply:
not sure when but i think it's at least theoretically possible: https://github.com/getsentry/relay/blob/04f531e6a0d0b3b65283520808f945b030a9457d/relay-kafka/src/producer/utils.rs#L110

Reply:
OK fair enough.